tests/expected.txt

```diff
@@ -23,7 +23,7 @@ examples/poster_881f4dc8dcf069e707bf61af95492d86        a34be06d22105b6c02394a9f
 examples/rl1/4up_b1c400de699af29ea3f1983bb26870ab                   e21dfdd9ae56ddb261dc3d02bf6da198
```

Comment:
Why skipping some tests below?

Reply:
They where not working and i could not make them work correctly. Can you try to run this?